cmd/cli/serve/util.go

```diff
@@ -47,94 +44,131 @@ func GetComputeConfig(
 		JobSelectionPolicy: node.JobSelectionPolicy{
-			Locality:            semantic.JobSelectionDataLocality(cfg.Node.Requester.JobSelectionPolicy.Locality),
-			RejectStatelessJobs: cfg.Node.Requester.JobSelectionPolicy.RejectStatelessJobs,
-			AcceptNetworkedJobs: cfg.Node.Requester.JobSelectionPolicy.AcceptNetworkedJobs,
-			ProbeHTTP:           cfg.Node.Requester.JobSelectionPolicy.ProbeHTTP,
-			ProbeExec:           cfg.Node.Requester.JobSelectionPolicy.ProbeExec,
+			// TODO(review): assumedly policy should be to accept jobs with data anywhere?
+			Locality:            semantic.Anywhere,
+			RejectStatelessJobs: cfg.JobAdmissionControl.RejectStatelessJobs,
+			AcceptNetworkedJobs: cfg.JobAdmissionControl.AcceptNetworkedJobs,
+			ProbeHTTP:           cfg.JobAdmissionControl.ProbeHTTP,
+			ProbeExec:           cfg.JobAdmissionControl.ProbeExec,
 		},
-		FailureInjectionConfig:         cfg.Node.Requester.FailureInjectionConfig,
-		EvalBrokerVisibilityTimeout:    time.Duration(cfg.Node.Requester.EvaluationBroker.EvalBrokerVisibilityTimeout),
-		EvalBrokerInitialRetryDelay:    time.Duration(cfg.Node.Requester.EvaluationBroker.EvalBrokerInitialRetryDelay),
-		EvalBrokerSubsequentRetryDelay: time.Duration(cfg.Node.Requester.EvaluationBroker.EvalBrokerSubsequentRetryDelay),
-		EvalBrokerMaxRetryCount:        cfg.Node.Requester.EvaluationBroker.EvalBrokerMaxRetryCount,
-		WorkerCount:                    cfg.Node.Requester.Worker.WorkerCount,
-		NodeOverSubscriptionFactor:     cfg.Node.Requester.Scheduler.NodeOverSubscriptionFactor,
-		WorkerEvalDequeueTimeout:       time.Duration(cfg.Node.Requester.Worker.WorkerEvalDequeueTimeout),
-		WorkerEvalDequeueBaseBackoff:   time.Duration(cfg.Node.Requester.Worker.WorkerEvalDequeueBaseBackoff),
-		WorkerEvalDequeueMaxBackoff:    time.Duration(cfg.Node.Requester.Worker.WorkerEvalDequeueMaxBackoff),
-		SchedulerQueueBackoff:          time.Duration(cfg.Node.Requester.Scheduler.QueueBackoff),
-		S3PreSignedURLExpiration:       time.Duration(cfg.Node.Requester.StorageProvider.S3.PreSignedURLExpiration),
-		S3PreSignedURLDisabled:         cfg.Node.Requester.StorageProvider.S3.PreSignedURLDisabled,
-		TranslationEnabled:             cfg.Node.Requester.TranslationEnabled,
-		JobStore:                       jobStore,
-		DefaultPublisher:               cfg.Node.Requester.DefaultPublisher,
-		NodeInfoStoreTTL:               time.Duration(cfg.Node.Requester.NodeInfoStoreTTL),
-	})
+		EvalBrokerVisibilityTimeout: time.Duration(cfg.Orchestrator.EvaluationBroker.VisibilityTimeout),
+		EvalBrokerMaxRetryCount:     cfg.Orchestrator.EvaluationBroker.MaxRetryCount,
+		WorkerCount:                 cfg.Orchestrator.Scheduler.WorkerCount,
+		TranslationEnabled:          cfg.FeatureFlags.ExecTranslation,
+		JobStore:                    jobStore,
+		// TODO(review): of the default job type configs it's not clear what we should be using here.
+		DefaultPublisher: cfg.JobDefaults.Batch.Task.Publisher.Type,
```

Comment:
@wdbaruni the current configuration allows the `DefaultPublisher` to be configured based on the job type. Should I implement the change to configure this in the requester now? Or wait and do it later as my previous comments have suggested?